src/test/java/org/candlepin/resource/EntitlementResourceTest.java

```diff
@@ -454,4 +454,18 @@ public void testUpdateEntitlementWithValidQuantityUpdate()
 
         verify(entitler).adjustEntitlementQuantity(nullable(Consumer.class), eq(entitlement), eq(4));
     }
+
+    @Test
+    public void testUpdateEntitlementWithInvalidEntitlementId() {
+        String entitlementId = "uniqueId";
+        EntitlementDTO entitlementDTO = new EntitlementDTO();
+        entitlementDTO.setId(entitlementId);
+        entitlementDTO.setQuantity(4);
+
+        doReturn(null).when(entitlementCurator).get(eq(entitlementId));
```

Comment:
You can omit this entirely for this test, as the default return from a mock is null.

Reply:
Good call! Updated